src/resources/js/v2/rsvp-ari.js

```diff
@@ -79,20 +79,20 @@ tribe.tickets.rsvp.ari = {};
 
 		// Set all forms as hidden.
 		$container.find( obj.selectors.guestFormFields ).addClass( obj.selectors.hiddenElement.className() );
-		$container.find( obj.selectors.guestFormFields ).attr( 'hidden', true );
+		$container.find( obj.selectors.guestFormFields ).prop( 'hidden', true );
 
 		// Show the selected guest.
 		obj.showElement( $targetGuestForm );
 		$targetGuestForm.removeAttr( 'hidden' );
 
 		// Set the classes for inactive.
 		$guestListButtons.addClass( obj.selectors.guestListItemButtonInactive.className() );
-		$guestListButtons.attr( 'aria-selected', false );
+		$guestListButtons.prop( 'aria-selected', false );
 
 		// Set the active class for the current.
 		const $targetGuestButton = $container.find( obj.selectors.guestListItemButton + '[data-guest-number="' + guestNumber + '"]' );
 		$targetGuestButton.removeClass( obj.selectors.guestListItemButtonInactive.className() );
-		$targetGuestButton.attr( 'aria-selected', true );
+		$targetGuestButton.prop( 'aria-selected', true );
```

Comment:
these should be `.attr( 'aria-selected', 'true' )` or `attr( 'aria-selected', 'false' )`, the value should be a string of `true` or `false`.

Reply:
@bordoni Does this still need to be addressed?